submissions/onyinyeChukwuka/javascript/contactform.js

```diff
@@ -0,0 +1,41 @@
+let fullName = document.getElementById('fullname');
```

Comment:
@Onyinye101 I think you should use the signup.js file since  most of the functions there are the same with the one here. You can change the name of the file to signup-contactform. Change the name of the file, where it has been used. Then move the necessary variable in there.